wire-protoc-compatibility-tests/src/test/java/com/squareup/wire/StructTest.kt

```diff
@@ -416,7 +414,6 @@ class StructTest {
       |}""".trimMargin()
     // Protoc doesn't print those unless explicitly set.
     val moshiJson = """{
-      |  "nullValue": null,
```

Comment:
I didn’t set out to make these deltas; they came out accidentally. Any concerns? I think probably the thing we want to be most deliberate about is when to omit null. (Always?)

Reply:
All good